_posts/2016-03-17-persistent-neglectedness.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  "Persistently neglected causes"
+date:   2016-03-17 11:58:26
+author: Michael Peyton Jones
+categories:
+---
+
+A heuristic that we often use when assessing causes is that of *neglectedness*.
+If a cause is neglected, then that means that it is not receiving an amount of
+attention commensurate to it's seriousness. Given that most causes suffer from
```

Comment:
80k's definition: 
Uncrowded: If we add more resources to the cause, we can expect more promising interventions to be carried out. Uncrowded causes are often undervalued or neglected by society. There may be a shortage of important actors within the cause.

https://80000hours.org/2013/12/a-framework-for-strategically-selecting-a-cause/